### ISSUE TITLE
Doesn't login

### ISSUE THREAD

**Issue by Duskfall** [JoeBili/bet365-websocket-crawler#2]
I don't seem be able to make it work, I just get the html page every time. Any thoughts? Why do you need the qq.com token?
![image](https://user-images.githubusercontent.com/2528125/85432780-5c28bd00-b57b-11ea-9c74-96f8f5911f9e.png)

Thanks

**Comment by JoeBili** [comment]
yes,  this code can'nt work now,  you can use my api.   I crawl data one wss one game, I sell API, real time data.   my telegram: https://t.me/JoeBili
